src/pages/Profile/EditProfile/EditProfile.ts

```diff
@@ -0,0 +1,131 @@
+import Block from "../../../utils/Block";
+import template from "../EditProfile/editProfile.hbs";
+import "../index.css";
+import {BackButton} from "../../../components/button/BackButton";
+import {Avatar} from "../../../components/avatar/Avatar";
+import {Name} from "../../../components/avatar/name/Name";
+import {Input} from "../../../components/input/Input";
+import {Button} from "../../../components/button/Button";
+
+export default class EditProfilePage extends Block {
+    data = {
+        email: "pochta@yandex.ru",
+        login: "ivanivanov",
+        firstName: "Иван",
+        secondName: "Иванов",
+        displayName: "Иван",
+        phone: "79099673030",
+    }
+
+    protected initChildren() {
+        this.children.backBtn = new BackButton({
+            id: "back-btn",
+            href: "/chats",
+        });
+        this.children.avatar = new Avatar({
+            avatarSrc: "",
+        });
+        this.children.nameTitle = new Name({
+            name: "Иван",
+        });
+        this.children.inputEmail = new Input({
+            id: "email",
+            type: "email",
+            label: "Email",
+            name: "email",
+            placeholder: "pochta@yandex.ru",
+            value: "pochta@yandex.ru",
+            readonly: false,
+            events: {
+                change: (e: any): void => {
+                    this.data.email = e.target.value;
+                }
+            }
+        });
```

Comment:
До этого у тебя строки заканчивались на `,` в таких случаях. И это хорошо делать, потому что положительно сказывается на диффах в системах контроля версий
```suggestion
            events: {
                change: (e: any): void => {
                    this.data.email = e.target.value;
                }
            },
        });
```